src/runtime/cache.ts

```diff
@@ -69,6 +71,7 @@ export function defineCachedFunction<T = any>(
     }
 
     const expired =
+      refresh ||
```

Comment:
I think it should be called? Also i'm thinking if we can give it a more generic name such as `skipCache(..)` as it is a passive checker (refresh seems like an active function that does refresh when called)

Reply:
It is being called but in the returned async function below following the same pattern as `getKey` :) As for the name, I may be wrong but I feel like `skipCache` implies it is just a one-time thing rather than a permanent cache override; what do you think?

Reply:
Right. I think it could be prefixed with `do` or `force` to clarify it is not same as option but a flag.

Agreed about skipCache naming. What about `shouldSkipCache` or `shouldInvalidateCache` or something similar?

Reply:
I like `shouldInvalidateCache`, I think it's clear enough!